.devcontainer/devcontainer.json

```diff
@@ -1,7 +1,7 @@
 {
 	"name": "Dapr Dev Environment",
 	// Update the container version when you publish dev-container
-	"image": "ghcr.io/dapr/dapr-dev:0.1.9",
+	"image": "ghcr.io/dapr/dapr-dev:latest",
```

Comment:
Now does it auto-build and push?

Reply:
https://github.com/dapr/dapr/actions/runs/3606497150/workflow#L16

Reply:
Although we have updated “build dev container” workflow, but for developers, they can not get those changes, right?

Could we re-enable this job?

Reply:
What do you mean? The job is enabled, just needs to be triggered manually

Reply:
Done. It now runs weekly too

Reply:
If its being triggered automatically, can this create an issue if we are always going to use the latest dev container?

Reply:
I think mostly developers would be using it, so may be that is better if we get latest.

Reply:
What kind of issues are you concerned with?

Reply:
Can we avoid using latest?